CHANGELOG.md

```diff
@@ -28,6 +38,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Changed DROP output column names to camel case [#132](https://github.com/genomic-medicine-sweden/tomte/pull/132)
 - Updated bcftools view and norm to be able to create index within the actual module [#137](https://github.com/genomic-medicine-sweden/tomte/pull/137)
 - Updated bcftools annot to be able to create index within the actual module [#140](https://github.com/genomic-medicine-sweden/tomte/pull/140)
+- Updated modules bcftools/merge, bcftools/mpileup, bcftools/stats, ensemblvep/vep, fastp, gawk, multiqc, salmon/quant, samtools/faidx, samtools/index and samtools/view [#141](https://github.com/genomic-medicine-sweden/tomte/pull/141)
```

Comment:
I see raredisease has a "Module updates" table in the CHANGELOG, which looks pretty nice imo, like the parameters table you had in the previous release.  

https://github.com/nf-core/raredisease/blob/master/CHANGELOG.md#module-updates

Reply:
I noticed in on your PR, it does look super nice, I will add it before merging